lib/rules/UseEffectRulePlugin.js

```diff
@@ -127,11 +167,12 @@ class UseEffectRulePlugin {
 				if (unhandledProperties.has("loader")) {
 					unhandledProperties.delete("loader");
 					unhandledProperties.delete("options");
-					unhandledProperties.delete("enforce");
 
-					const loader = rule.loader;
+					/** @typedef {WebpackOptionsRuleSetLoader} */
+					const loader =
+						/** @type {WebpackOptionsRuleSetLoader} */ (rule.loader);
```

Comment:
no support for `RuleSetUse` ?
https://github.com/webpack/webpack/blob/next/declarations/WebpackOptions.d.ts#L628